cdap-spark-core-base/src/k8s/Dockerfile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright © 2021 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+# Creates a customized Spark image that can be used to run CDAP Spark programs
+# This should be run from the CDAP base directory.
+# Before creating the image, cdap-spark-core and cdap-kubernetes must be built:
+#
+# mvn package -P dist,k8s-dist -pl cdap-spark-core3_2.12,cdap-kubernetes -am -DskipTests
+#
+# Then build the image:
+#
+# docker build --no-cache --build-arg base_image=<image> \
+#   --build-arg cdap_version=`mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec` \
+#   -f cdap-spark-core-base/src/k8s/Dockerfile \
+#   . -t <image>:<tag>
+
+ARG base_image
+
+FROM ${base_image}
+
+ARG cdap_spark_module=cdap-spark-core3_2.12
+ARG cdap_version
+
+# copy cdap-spark-core and its dependencies
+COPY ${cdap_spark_module}/target/k8s/${cdap_spark_module} /opt/cdap/cdap-spark-core
```

Comment:
I would suggest have the maven to output to `${cdap_spark_module}/target/k8s`. We shouldn't need the `${cdap_spark_module}` suffix again

Reply:
👍